_includes/files.html

```diff
@@ -7,22 +7,44 @@
   displayed inside a pre block and include a download link underneath.
 
   @param {string} "title" - the yml meta field for title in the download file
+  @param {string} "language" - the coding language used for syntax highlighting
 
 -->
 {% endcomment %}
 
+{% comment %}
+<!-- determine correct collection based on language param -->
+{% endcomment %}
+{% if include.language == "yml" or include.language == "yaml" %}
+	{% assign files = site.ymls | where: 'title', include.title %}
+{% else %}
+	{% assign files = site.files | where: 'title', include.title %}
+{% endif %}
+
+{% comment %}
+<!-- determine syntax highlighting language.  Defaults to bash -->
```

Comment:
Doesn't it just default to no highlighting language? Blank triple backticks is my desired behavior and it looks like that's what the code is doing, so it's probably just a matter of updating the comment.